packages/paste-website/scripts/fetch-data.mjs

```diff
@@ -0,0 +1,115 @@
+import * as fs from 'fs/promises';
```

Comment:
praise: I love this idea to prebuild this file for the website launch from the Airtable source data.